### PR TITLE
Provide a mechanism to customise aspects of the generated Slack messages.

### PR DESCRIPTION
**Pull request by eddgrant**
Hey folks,

Thanks for creating this repo :pray: , we're starting to use it over at HMRC Digital.

I've been discussing with @m-green that we'd like to be able to customise various aspects of the Slack message, to that end this change makes it possible to customise the prefix, Slack username and icon emoji of generated Slack messages. It defaults to the existing hard-coded settings so I'm anticipating that it should be regression-free for GDS users, however opens up the ability to customise things for other users.

Most grateful if you'd be willing to give it a look and consider it for inclusion, happy to have a chat if there are any questions/ concerns/ things that you'd like to see addressed for inclusion etc.

Many thanks,

Edd

### CODE REVIEW

**Comment by eddgrant** [comment]
Hey folks,

Just giving this a gentle nudge, would love to hear your thoughts on whether you'd consider this for inclusion.

Many thanks,

Edd

**Comment by eddgrant** [comment]
Hey @m-green , @lfdebrux ,

Hope you'll excuse my pinging you directly. Was wondering if either of you are still working on this repository? Would be great to chat about this request if you are.

Many thanks,

Edd

File: repository.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+repoVisibility: public_0C3F0CE3E6E6448FAD341E7BFA50FCD333E06A20CFF05FCACE61154DDBBADF71
```
Comment (lfdebrux) [inline repository.yaml:1]: What is this for? Is it required?

Comment (eddgrant) [reply repository.yaml:1]: Sorry that's an HMRC construct that we use in all our public repos. Not required here, will remove.

Comment (eddgrant) [reply repository.yaml]: Removed.

**Review by lfdebrux — APPROVED** [review]

**Pull request merged**